src/Draco.Compiler/Internal/Symbols/Source/SourceModuleSymbol.cs

```diff
@@ -24,13 +26,16 @@ internal sealed class SourceModuleSymbol : ModuleSymbol, ISourceSymbol
     public override Symbol? ContainingSymbol { get; }
     public override string Name => this.declaration.Name;
 
-    public override SyntaxNode? DeclaringSyntax => null;
+    public override SymbolDocumentation Documentation => InterlockedUtils.InitializeNull(ref this.documentation, () => new MarkdownDocumentationExtractor(this.RawDocumentation, this).Extract());
+    private SymbolDocumentation? documentation;
 
     /// <summary>
     /// The syntaxes contributing to this module.
     /// </summary>
     public IEnumerable<SyntaxNode> DeclaringSyntaxes => this.declaration.DeclaringSyntaxes;
 
+    private string RawDocumentation => this.DeclaringSyntaxes.SingleOrDefault()?.Documentation ?? string.Empty;
```

Comment:
This can potentially throw, at least mitigate by first filtering for syntaxes that _have_ docs, then assume there's a single one of those.